build/q.css

```diff
@@ -341,6 +341,8 @@ svg, :root {
 	max-width: 100%;
 	overflow-x: auto;
 	font-family: var( --Q-font-family-sans );
+	/*letter-spacing: 0.03em;*/
```

Comment:
we can remove code that we are not using, we you are still experimenting and will need to uncomment this in future